jwst/tweakreg/wcsimage.py

```diff
@@ -1296,9 +1275,14 @@ def __init__(self, catalog, name=None):
         name : str, None, optional
             Name of the reference catalog.
 
+        footprint_tol : float, optional
+            Matching tolerance in arcsec. This is used to estimate catalog's
```

Comment:
The `imalign` doc string says tolerance is in units of pixels. Is this an inconsistency or is this a different tolerance?

Reply:
@hbushouse It is inconsistency (pixels were used in the old tweakreg). `spec` however, was documented correctly: https://github.com/spacetelescope/jwst/blob/d7daa6c9cc60fc9870add2950ca06a424056db28/jwst/tweakreg/tweakreg_step.py#L45

Reply:
👍